visual_behavior/utilities.py

```diff
@@ -698,3 +698,56 @@ def event_triggered_response(df, parameter, event_times, time_key=None, t_before
         melted['event_number'] = melted['variable'].map(lambda s: s.split('event_')[1].split('_')[0])
         melted['event_time'] = melted['variable'].map(lambda s: s.split('t=')[1])
         return melted.drop(columns=['variable']).rename(columns={'value': parameter})
+
+
+def annotate_licks(dataset, inplace=False, lick_bout_ili=2):
```

Comment:
Why default the lick_bout_ili to 2 seconds? I think I default it to 750ms. I'm open to changing definitions but I think we should be consistent.

Reply:
Good point. Your default in the GLM is 0.7:
https://github.com/AllenInstitute/visual_behavior_glm/blob/86a816ebc4dcc8d450f83f77611c2ee7afdf7b93/visual_behavior_glm/GLM_params.py#L182

I matched that.